plugin/execute_command.py

```diff
@@ -0,0 +1,56 @@
+import sublime
+from .core.registry import client_for_view, LspTextCommand
+from .core.settings import client_configs
+from .core.protocol import Request
+from .core.logging import debug
+from .core.rpc import Client
+
+try:
+    from typing import List, Optional, Dict, Any
+    assert List and Optional and Dict and Any
+except ImportError:
+    pass
+
+
+class LspExecuteCommand(LspTextCommand):
+    def __init__(self, view):
+        super().__init__(view)
+
+    def run(self, command_name) -> None:
+        client = client_for_view(self.view)
+        if not client:
+            self.view.show_popup("No connection with server")
```

Comment:
Just return silently here. There should always be a client available. This command is not visible from the command palette if there is no ClientConfig present for the view.

Reply:
ok but do you know how to load the config only for clients that are Initialized ?
If I have client A and B in my settings but only A I have a current session, how can I get only configs of A ?